### ISSUE TITLE
Custom Heading Style font defaults to Calibiri only for Arial

### ISSUE THREAD

**Issue by monoaminergic** [stevencohn/OneMore#1334]
# Problem to Solve
- Changing the font of a heading to Arial results in any text it's applied to defaulting to Calibri. 
- This occurs to all styles in the _"Heading - include in TOC"_ category,
- Other categories I have tested (e.g., _"Quote"_) are successful in changing the font to Arial.
- Other fonts (e.g., Arial Narrow) can be changed successfully for the _"Heading - include in TOC"_ category.

This isn't a major issue for me personally, I just wanted to make sure you know :)

**Comment by stevencohn** [comment]
Unable to reproduce. Do you have a set of steps that reliably demonstrate the problem?

**Comment by nvn-pro** [comment]
I had the same issue, I changed all the fonts for all Headings in My Styles to Arial and changed the font size. When I applied these modified Headings to text container, sometimes the modified Headings are applied correctly with Arial and correct font size, but sometimes it still keep Calibri and the same font size as default. The issue is that sometimes it is OK, sometimes it not OK.
Let's talk about the case when it is not applied correctly, I try to put the cursor at the end of that paragraph, then I click the Format Painter and apply the format painter to this paragraph itself, then this paragraph become OK, with correct Arial and font size.
